README.md

```diff
@@ -1 +1,169 @@
+
+   ```bash
+   git clone https://github.com/axcell-0/school-library.git
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd school-library
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Usage <a name="usage"></a>
+
+## Adding Students and Teachers <a name="adding-students-and-teachers"></a>
+
+1. Run the following command to start the app:
+
+   ```bash
+   ruby main.rb
+   ```
+
+2. Follow the on-screen prompts to add new students or teachers to the library's database.
+
+## Adding Books <a name="adding-books"></a>
+
+1. Launch the app:
+
+   ```bash
+   ruby main.rb
+   ```
+
+2. Choose the option to add new books and provide the necessary book details.
+
+## Borrowing Records <a name="borrowing-records"></a>
+
+1. Start the app:
+
+   ```bash
+   ruby main.rb
+   ```
+
+2. Navigate to the borrowing records section and follow the prompts to track book borrowing activities.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👤 Author <a name="author"></a>
+
+👤 Thierry Noumbuh
+
+- GitHub: [@kit0-0](https://github.com/axcell-0)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement User Authentication**: Allow users to create accounts, log in, and personalize their library experience.
+
+- [ ] **Search and Filter Functionality**: Add search and filter options to make it easier for users to find books in the library.
+
+- [ ] **Integration with External APIs**: Enhance the library's catalog by integrating with external book databases to provide detailed book information.
+
+- [ ] **User Ratings and Reviews**: Enable users to rate and review books, helping others make informed choices.
+
+- [ ] **Mobile-Friendly Design**: Optimize the application's design for mobile devices to provide a seamless experience on smartphones and tablets.
+
+- [ ] **Notification System**: Implement a notification system to alert users about due dates, reservations, and library announcements.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+# Contributing <a name="contributing"></a>
+
+We welcome contributions from the community. If you'd like to contribute, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature/fix.
+3. Make your changes and commit them.
+4. Push your branch to your forked repository.
+5. Open a pull request.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Support <a name="support"></a>
+
+If you encounter any issues or have questions about using the app, feel free to [contact our support team](https://github.com/axcell-0/school-library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# License <a name="license"></a>
+
+This project is licensed under the [MIT License](./LICENSE).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] It would be nice if you add the `Acknowledgement` section in your readme file to make your readme descriptive and professional 👍